README.md

```diff
@@ -59,20 +151,364 @@ of these challenges.
+to summarise the ***system*** we are building. ([`#HelpWanted`]())
+Instead we can describe the App as a means 
+to answer the following **10 questions**:
+
+1. **What** are my **goals** in life? 
+(personal/professional/family/financial/fitness/etc.)
+2. Is what I'm doing **right now** 
+moving me **towards** one of my **goals**?
+3. What is the **highest priority task** 
+that I _should_ be working on **right now**?
+4. Is the **highest priority task** well defined 
+or does it need more detail before I can start on it?
+5. What is the **_smallest_ thing** I can **do** 
+to **make progress** toward one of my **goals**?
+6. Am I **_effectively_ communicating** what I'm working on
+with other people so they know when I am **focussing**?
+7. Is there anything **requiring my attention right now** 
+that **_cannot_ wait** until the end 
+of my current highest priority task 
+or focus time interval?
+8. **Who** in my team/family needs me 
+in order to make progress 
+on their current (or next) task?
+9. Which task on _my_ list **can I _delegate_**
+to someone else so that I can focus on 
+the item that _only_ I can do.
+10. What should I **_avoid_ doing** 
+or **say** "**no**" to now
+so that I can **focus** on my highest priority item?
+
+<br />
+
+## "_The difference between successful people and really successful people is that really successful people say no to almost everything._" ~ [Warren Buffet](https://medium.com/the-ascent/you-can-do-amazing-things-but-only-if-you-have-time-to-do-them-31c07ab5b04e)
+
+What do _you_ need to (_politely and plainly_) say "***no***" to 
+so you can **focus** on your **goals**?
+
+
+<!-- 
+https://zenhabits.net/enough-time
+https://www.lifehack.org/articles/productivity/10-reasons-why-some-people-feel-like-they-dont-have-enough-time.html
+-->
+
+<br />
+
+## 3 Stages of Our Roadmap
+
+
+### 1. _Individual_ Person Experience -> [_Single Player_](https://en.wikipedia.org/wiki/Single-player_video_game)
+
+If the App is not _immediately_ useful to an _individual_ person 
+and relies on "network effects" it will be 
+[DOA](https://en.wikipedia.org/wiki/Dead_on_arrival).  <br />
+We need to _optimise_ for the individual _first_ 
+and then once we are able to use the App individually _ourselves_, 
+we can move on to the "Team" experience (see below). 
+
+In the _individual_ person experience, the person uses the dwyl App to:
+
++ **capture** > **brain-dump** everything they have in their head 
+so that they can think clearly (_without a cluttered mind_) and prioritise. 📝 
+  
+  > Capturing text (_and later voice memos and images_) as "items" 
+  which can then be categorised/organised using tags 
+  is the most intiuitive and versatile way of storing data.
+  We only need _one_ field or data type (`text`).
+  We can then apply `tags` to an `item` to make it into _anything_ else.
+  And then the UI renders the item with the appropriate UX 
+  for an `item` with the `tag` `"task"`, display a checkbox next to it.
```

Comment:
```suggestion
  for an `item` with the `tag` `"task"`, displaying a checkbox next to it.
```

Reply:
This is technically acceptable grammar; the action is to "_display_" the checkbox.